examples/handlebars_templates/src/main.rs

```diff
@@ -24,13 +26,31 @@ fn index() -> Redirect {
 }
 
 #[get("/hello/<name>")]
-fn get(name: String) -> Template {
+fn hello(name: String) -> Template {
+    let page = "index".to_string();
+    let title = format!("Rocket Example - {}", page).to_string();
```

Comment:
Why not pass in `"index"` as the template context and then render it as `"Rocket Example - {}"` in the template?